.storybook/config.js

```diff
@@ -16,12 +18,19 @@ function loadStories() {
   req.keys().forEach(filename => req(filename));
 }
 
-addDecorator(
-  centered,
-  withOptions({
-    name: 'React Containers',
-    url: 'https://github.com/zendeskgarden/react-containers'
-  })
-);
+addDecorator(centered);
+addDecorator(withA11y);
+
+addParameters({
+  options: {
+    theme: create({
+      brandTitle: 'React Containers',
+      brandUrl: 'https://github.com/zendeskgarden/react-containers',
+      brandImage: 'https://garden.zendesk.com/favicon.ico',
```

Comment:
Are we able to not hot-link the favicon? I think there is a way to host static files within Storybook